### ISSUE TITLE
Group label viewed  as tag  when group is selected

### ISSUE THREAD

**Issue by freddidierRTE** [sa-si-dev/virtual-select#134]
When using group option and user select the group , it show up in  the input field : 

![image](https://user-images.githubusercontent.com/58251627/163940859-e86d3496-0321-4c65-8091-890915d8ca68.png)

Would it be possible to not show the tag for the group ?  (group 1 on the example)

Furthermore , you cannot remove the group using the cross on the tag.

I use option `useGroupValue` but it does not change the behaviour .

My code example : 

```
<html>

<head>
    <link rel="stylesheet" href="virtual-select.min.css">
</head>

<body>

    <script src="./virtual-select.min.js"></script>

    <div id="test-select"></div>

    <br />

    <script>
        VirtualSelect.init({
            ele: '#test-select',
            options: [
                {
                    label: 'Groupe 1',
                    options: [
                        {label: 'Service A', value: 'SA'},
                        {label: 'Service B', value: 'SB'},
                        {label: 'Service C', value: 'SC'}
                    ]
                },
                {
                    label: 'Groupe 2',
                    options: [
                        {label: 'Service D', value: 'SD'},
                        {label: 'Service E', value: 'SE'},
                        {label: 'Service F', value: 'SF'}
                    ]
                }
            ],
            multiple: true,
            showValueAsTags: true,
            useGroupValue: false,
            placeholder: ""
        });
        document.querySelector('#test-select').addEventListener('change', function() {
                console.log("Selected = ", document.querySelector('#test-select').value);
              });
    </script>
</body>

</html>
```

**Comment by sa-si-dev** [comment]
@freddidierRTE thanks for letting me know it. I will fix it in the future release.

**Comment by sa-si-dev** [comment]
@freddidierRTE This issue has been fixed in the [release v1.0.29](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.29).
